source/_patterns/03-organisms/header/demo/headers.twig

```diff
@@ -0,0 +1,6 @@
+<header role="banner">
+  {% embed '@organisms/header/_header.twig' %}
+    {% block branding %}
```

Comment:
Doing this to wipe out the branding for the header demo?

Reply:
Thought here was to leave it following the pattern we use elsewhere to suggest it can be overridden. That said, I'm still not totally understanding the little differences between include, extends and embed (other than embed is combining include + extends?). Plan on putting some time into reading about it today since a lot of this branch could have been easier with that knowledge.

Reply:
Having the empty block here in this implementation blows away whatever lives by default in this twig blog within the `_header.twig` file (which is the branding atom).